tests/build.go

```diff
@@ -106,7 +107,7 @@ func testImageIDOutput(t *testing.T, sb integration.Sandbox) {
 
 	cmd := buildxCmd(
 		sb,
-		withArgs("build", "-q", outFlag, "--iidfile", filepath.Join(targetDir, "iid.txt"), "--metadata-file", filepath.Join(targetDir, "md.json"), dir),
+		withArgs("build", "-q", "--provenance", "false", outFlag, "--iidfile", filepath.Join(targetDir, "iid.txt"), "--metadata-file", filepath.Join(targetDir, "md.json"), dir),
```

Comment:
@tonistiigi Needs to disable provenance for this test with containerd storage as it would export an OCI image:

```json
{
  "containerimage.buildinfo": {
    "frontend": "dockerfile.v0",
    "attrs": {
      "filename": "Dockerfile"
    },
    "sources": [
      {
        "type": "docker-image",
        "ref": "docker.io/library/busybox:latest",
        "pin": "sha256:0d5a701f0ca53f38723108687add000e1922f812d4187dea7feaee85d2f5a6c5"
      }
    ]
  },
  "containerimage.descriptor": {
    "mediaType": "application/vnd.oci.image.index.v1+json",
    "digest": "sha256:ce51e4ac359feabf7a34388bb599e84873ba9299ce8b863ead9a093a16ffc875",
    "size": 855
  },
  "containerimage.digest": "sha256:ce51e4ac359feabf7a34388bb599e84873ba9299ce8b863ead9a093a16ffc875",
  "image.name": "moby-dangling@sha256:ce51e4ac359feabf7a34388bb599e84873ba9299ce8b863ead9a093a16ffc875"
}
```

This is a test case we can add in follow-up. I guess we need to read `"containerimage.descriptor".annotations."config.digest"`.